src/background.js

```diff
@@ -252,20 +305,41 @@ const rollout = {
     // Determine to skip additional heuristics (by presence of an enterprise policy)
     if (policyEnableDoH === "no_policy_set") {
       // Resetting skipHeuristicsCheck in case a user had a policy and then removed it!
-      this.setSetting("skipHeuristicsCheck", false);
+      this.setSetting("doh-rollout.skipHeuristicsCheck", false);
     } else {
       // Don't check for prefHasUserValue if policy is set to disable DoH
-      this.setSetting("skipHeuristicsCheck", true);
+      this.setSetting("doh-rollout.skipHeuristicsCheck", true);
     }
     return;
 
   },
 
+  async migrateLocalStoragePrefs() {
+    if (await this.getSetting("doneFirstRun")){
+      await this.setSetting("doh-rollout.doneFirstRun");
+    }
+
+    if (await this.getSetting("skipHeuristicsCheck")){
+      await this.setSetting("doh-rollout.skipHeuristicsCheck");
+    }
+
+    // Set pref to skip this function in the future.
+    browser.experiments.preferences.setBoolPref(DOH_BALROG_MIGRATION_PREF, true);
+
```

Comment:
nit: extra newline

Reply:
Updated this request in a seperate PR – #175